tests/import/test_import_cli.py

```diff
@@ -164,35 +161,44 @@ def test_max_candidates_custom(self, tmp_config):
             tmp_db=True,
         )
 
-        mock_new_album = Mock()
         mock_candidates = [Mock() for _ in range(10)]
 
         with patch(
             "moe.moe_import.import_cli.candidate_prompt", autospec=True
         ) as mock_candidate_prompt:
             config.CONFIG.pm.hook.process_candidates(
-                new_album=mock_new_album,
+                new_album=ANY,
                 candidates=mock_candidates,
             )
 
-        mock_candidate_prompt.assert_called_once_with(
-            mock_new_album, mock_candidates[:3]
-        )
+        mock_candidate_prompt.assert_called_once_with(ANY, mock_candidates[:3])
 
     def test_max_candidates_more_than_available(self):
         """Should handle cases where max_candidates > available candidates."""
-        mock_new_album = Mock()
         mock_candidates = [Mock() for _ in range(2)]
 
         with patch(
             "moe.moe_import.import_cli.candidate_prompt", autospec=True
         ) as mock_candidate_prompt:
             config.CONFIG.pm.hook.process_candidates(
-                new_album=mock_new_album,
+                new_album=ANY,
```

Comment:
Sorry for leading you astray here, but `ANY` should only be used for assertions, I don't think it's right to use it for passing values. `new_album=Mock()` would be more appropriate, but leave `ANY` in the assertion check. Same goes for the other tests.

Reply:
Ah thanks for the explanation! This is also pointed out in the docs: https://docs.python.org/3/library/unittest.mock.html#any

Fixed!